db/mycroft/metric_schema/tables/device_activity.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE metric.device_activity (
+    id              uuid        PRIMARY KEY DEFAULT gen_random_uuid(),
+    activity_dt     date        NOT NULL DEFAULT current_date,
+    platform         text        NOT NULL,
+    paired          INTEGER     NOT NULL DEFAULT 0,
+    connected       INTEGER     NOT NULL DEFAULT 0,
```

Comment:
Are these: 
- "is the device paired/connected?"
- "how many times has the device paired / connected?"

Or is it for an account "how many devices are paired/connected?"